test/jdk/java/net/vthread/BlockingSocketOps.java

```diff
@@ -59,6 +59,7 @@
 import java.net.SocketTimeoutException;
 
 import jdk.test.lib.thread.VThreadRunner;
+import jdk.test.lib.Platform;
```

Comment:
This seems to be left over from one of your previous iterations.

Reply:
Thanks for catching that. I made this change.